packages/trader/src/RangeTrader.ts

```diff
@@ -165,6 +169,13 @@ export class RangeTrader {
       .mul(this.fixedPointAdjustment) // Scale the numerator before division
       .div(this.normalizePriceFeedDecimals(expectedValue));
   }
+
+  // TODO: there are a number of places through the repo that re-use this method. it should be refactored to a common util.
+  _invertPriceSafely(price: BigNumber) {
```

Comment:
+1 to this comment, maybe add it as a common util in this PR and we can refactor it elsewhere as we go?

Reply:
yup! there are a few places of refactoring I want to do like this. I think we have a task tracking it for next sprint